pkg/regionpicker/gcp.go

```diff
@@ -0,0 +1,17 @@
+package regionpicker
+
+const (
+	CloudGCP CloudName = "gcp"
+)
+
+type GCP struct{}
+
+// Regions returns a list of all known-gcp regions. This should not be used
+// over native GCP region fetching.
```

Comment:
this is pretty ugly for now (incomplete).  I'd prolly have users pass in a list for now, if this won't do discovery for them.